src/components/common/DataTable/columns.tsx

```
@@ -512,24 +512,25 @@ export const bookingColumns: ColumnDef<Booking>[] = [
           };
           break;
         default:
+          const statusStr = String(status || 'unknown');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)